app/styles/tracker.scss

```diff
@@ -199,41 +199,57 @@ info {
   }
 
   .info-footer {
-    @include transition(all .1s ease-out);
+    @include flexbox();
 
-    display: block;
-    margin-top: auto;
-    padding: 20px;
-    background-color: $brand-secondary-dark;
-    color: #fff;
-    text-align: center;
-    font-family: $font-secondary;
-    font-size: 11px;
-    font-weight: 700;
-    text-transform: uppercase;
-    letter-spacing: 1px;
-    text-decoration: none;
+    border-top: 2px solid #12345f;
 
-    .fa {
+    a {
       @include transition(all .1s ease-out);
 
-      font-size: 10px;
-      margin-left: 3px;
+      width: 50%;
+      padding: 20px;
+      background-color: darken($brand-secondary, 4%);
+      color: #fff;
+      font-family: $font-secondary;
+      font-size: 11px;
+      font-weight: 700;
+      text-transform: uppercase;
+      letter-spacing: 1px;
+      text-decoration: none;
+      text-align: center;
+
+      &:last-child {
+        background-color: $brand-secondary-dark;
```

Comment:
is it weird that the two buttons are different colors? when it first loaded, i kinda thought that the serebii link was already "selected" or something. do you not think so?